src/modules/prpn/PRPN.py

```diff
@@ -0,0 +1,142 @@
+'''
+Code for Parsing-Reading-Predict Networks (PRPN; Shen et al., 2018)
+Modules re-used from: https://github.com/yikangshen/PRPN
```

Comment:
Same comment as above. It should be clear which lines are exactly their code, and which lines are your edits of their code.

Reply:
And same in other similar files.

Reply:
This still seems a bit vague, which is really, really dangerous. How about something like: 'This file is a version of a class from [URL], modified to integrate with jiant.'

Reply:
The other files look good, though!